### PR TITLE
Updated gherkin interface in order to update table with placholder of exemple for scenario outlines

### PR DESCRIPTION
**Pull request by medtoure18**
Hello,
With this PR when you create a gherkin file that combine a scenerio outline and Table: Example value will be replaced inside the table.
I added a new test : IncludeExamplesInDataTable.feature

### CODE REVIEW

File: test/data/sandbox/features/step_definitions/my_steps.js [hunk 1]
```diff
@@ -47,3 +58,5 @@ After((test) => {
 Fail(() => {
   console.log(`-- failed ${test.title} --`);
 });
+
```
[the remark this answers is not in the document]
Comment (medtoure18) [reply test/data/sandbox/features/step_definitions/my_steps.js]: fixed

File: test/data/sandbox/features/step_definitions/my_steps.js [hunk 2]
```diff
@@ -34,7 +34,18 @@ Then('I should see overall price is "{float}" $', (price) => {
 Given('I login', () => {
   I.login('user', secret('password'));
 });
+Given(/^I have this product in my cart$/, (table) =>{
```
[the remark this answers is not in the document]
Comment (medtoure18) [reply test/data/sandbox/features/step_definitions/my_steps.js]: fixed

File: lib/interfaces/gherkin.js [hunk 3]
```diff
@@ -73,7 +73,7 @@ module.exports = (text) => {
           }
           const tags = child.tags.map(t => t.name);
           const title = `${child.name} ${JSON.stringify(current)} ${tags.join(' ')}`.trim();
-          const test = new Test(title, async () => runSteps(exampleSteps));
+          const test = new Test(title, async () => runSteps(addExampleInTable(exampleSteps,current)));
```
[the remark this answers is not in the document]
Comment (medtoure18) [reply lib/interfaces/gherkin.js]: fixed

**Comment by DavertMik** [comment]
Wow, thank you!
Looks very useful!

**Pull request merged**